python/GafferSceneUI/_SceneViewInspector.py

```diff
@@ -192,7 +195,8 @@ def setVisible( self, visible ) :
 
 	def __attachToView( self, sceneView ) :
 
-		# We add-in plugs to manage the our visibility as per grid/gnomon
+		# We add plugs to manage out visibility as per grid/gnomon. This plug is the
```

Comment:
out -> our

Reply:
I don't know what is wrong with my brain, sorry, fixed [here](https://github.com/GafferHQ/gaffer/pull/3832/commits/540625f6b4452ad4f3dec615e43566cadf0b2bf8).